packages/world/test/Bed.t.sol

```diff
@@ -120,9 +114,9 @@ contract BedTest is BiomesTest {
   }
 
   function testSleepFailsIfNotInPlayerInfluence() public {
-    (address alice, , VoxelCoord memory coord) = setupAirChunkWithPlayer();
+    (address alice, , Vec3 coord) = setupAirChunkWithPlayer();
 
-    VoxelCoord memory bedCoord = VoxelCoord(coord.x - 500, coord.y, coord.z);
+    Vec3 bedCoord = coord + vec3(500, 0, 0);
```

Comment:
intentional that `-500` turned into `+500`?

Reply:
it's the same for this test, we just want to be far away, we could probably improve it by setting the min distance required for it to be out of range